core-primitives/stf-interface/src/system_pallet.rs

```diff
@@ -25,3 +27,23 @@ pub trait SystemPalletAccountInterface<State, AccountId> {
 	/// Get the account date for a given account and state.
 	fn get_account_data(state: &mut State, account: &AccountId) -> Self::AccountData;
 }
+
+/// Interface trait of the system pallet for event specific interactions.
+pub trait SystemPalletEventInterface<State> {
+	type EventRecord;
+	type EventIndex;
+	type BlockNumber;
+	type Hash;
+
+	/// Get a Vec of bounded events.
+	fn get_events(state: &mut State) -> Vec<Box<Self::EventRecord>>;
```

Comment:
Can you add some new lines between the function names and the comment of the following functions?